### DISCUSSION TITLE
【分享】H.D.D实用软件！

### DISCUSSION THREAD

**Discussion by capacitor1** [share121/inter-knot#1447] [其他]
最新！重磅消息！为了帮助绳匠更好的使用H.D.D[^1]，更好的调教Fairy，我向各位绳匠分享一些萝卜：
**详细信息如下：**
everything：文件搜索，可在空洞中快速查找需要的文件，比H.D.D 10 / 11自带的要好用很多！
[Everything-1.4.1.1022.x64.Lite-Setup.zip](https://github.com/user-attachments/files/16573338/Everything-1.4.1.1022.x64.Lite-Setup.zip)

MediaInfo：多媒体元数据查看器，在录像店极其有用！有时候录像播放不了，可以使用此工具查看录像的编码和可能的错误，帮助确定录像播不了的原因
[MediaInfo_GUI_21.03_Windows.zip](https://github.com/user-attachments/files/16573339/MediaInfo_GUI_21.03_Windows.zip)

TrafficMonitor：网络实时流量查看器，帮助绳匠检测Fairy到底跑了多少流量！避免网费电费翻倍的情况
[TrafficMonitor.zip](https://github.com/user-attachments/files/16573340/TrafficMonitor.zip)

TrID：文件类型扫描，对于从空洞中获取的新数据，但却打不开，就可以使用TrID扫描一下来确定真正的格式！
[TrID.zip](https://github.com/user-attachments/files/16573341/TrID.zip)

[^1]:指 微软视窗操作系统（Windows）

**Comment by lllliufeng** [comment]
感谢lz，这就去跟空洞对线

**Comment by 290678** [comment]
插个idm下载器【激活】https://www.bilibili.com/read/cv34832931/
【下载idm】https://www.52pojie.cn/thread-1929729-1-1.html
